doc/api/module.md

```diff
@@ -28,8 +28,6 @@ added:
 A list of the names of all modules provided by Node.js. Can be used to verify
 if a module is maintained by a third party or not.
 
-Note: the list doesn't contain [prefix-only modules][] like `node:test`.
```

Comment:
Can you add the entry in the YAML comment please?

Reply:
Agree on this one